### PR TITLE
Model upgrade

### PR DESCRIPTION
**Pull request by MLazarevic36**

### CODE REVIEW

**Review by coka — CHANGES_REQUESTED** [review]
I'm seeing a lot of mixed `snake_case` and `camelCase` for column names. Please choose one or the other.

**Comment by MLazarevic36** [comment]
I pushed migrations, models and seeders changes, since I need live assistance on retrieval of data from database.

File: src/database/migrations/20191217081018-create-call.js [hunk 1]
```diff
@@ -8,18 +7,16 @@ module.exports = {
...
         autoIncrement: true,
         primaryKey: true
-        
       },
       created_at: {
         type: Sequelize.DATE,
         allowNull: false,
       },
-      call_data: {
+      number_of_call: {
```
Comment (coka) [inline src/database/migrations/20191217081018-create-call.js]: `call_number`

File: src/database/migrations/20191225101900-create-end-of-call-state.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('End_of_call_states', {
```
Comment (coka) [inline src/database/migrations/20191225101900-create-end-of-call-state.js]: I understand that you were going for a translation of "stanje", but "state" has a very well-understood meaning in programming, so I think avoiding usage of the word "state" in our schema. Maybe something like `Call_resolution_type`?

File: src/database/migrations/20191225105610-create-age.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Ages', {
```
Comment (coka) [inline src/database/migrations/20191225105610-create-age.js]: Let's hold off on creating this table until the requirements around age are more clearly defined.

File: src/database/migrations/20191225105719-create-number-of-calls.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Number_of_calls', {
```
Comment (coka) [inline src/database/migrations/20191225105719-create-number-of-calls.js]: `Call_number`, since this is effectively a friendly ID for the call.

**Review by coka — CHANGES_REQUESTED** [review]
Squash all of these migrations in a single migration file. They are one unit of work, and one effective change / diff of the schema. You can refer to #49 to see which issues cropped up in the previous attempt.

**Comment by coka** [comment]
> I pushed migrations, models and seeders changes, since I need live assistance on retrieval of data from database.

Sure thing. :+1: We can have a look after the daily.

File: src/database/migrations/20191225105719-create-number-of-calls.js (hunk 4 above)
Comment (MLazarevic36) [reply src/database/migrations/20191225105719-create-number-of-calls.js]: I think that Number_of_calls refers to how many times has caller called

File: src/database/migrations/20191225101900-create-end-of-call-state.js (hunk 2 above)
Comment (MLazarevic36) [reply src/database/migrations/20191225101900-create-end-of-call-state.js]: Sure

**Comment by MLazarevic36** [comment]
Should I squash seeders like migrations?

**Comment by coka** [comment]
> Should I squash seeders like migrations?

Yes, I think so. 🤔

**Review by coka — APPROVED** [review]

**Pull request merged**